server/session/session.go

```diff
@@ -232,6 +235,7 @@ func (s *Session) Close() error {
 // close closes the session, which in turn closes the controllable and the connection that the session
 // manages.
 func (s *Session) close() {
+	_ = s.viewLayer.Close()
```

Comment:
Why should one viewer leaving close the entire view layer?

Reply:
Because they left?

Reply:
what if multiple sessions has the same view layer

Reply:
1 session = 1 view layer

Reply:
Then why do view layers store multiple sessions at the moment?

Reply:
every player has it's own layer, so that each player can be different, only to a certain player